Added Notepad++ documentation

adamspihlman opened the pull request:
#Advanced HW 6 submission

Closes #152 

Name: Adam Spihlman
Uniqname: spihlman

## Description of Changes

Added description and features of Notepad++ in the editors/ directory. Also included link to download page for users who want to try out Notepad++

samkhn commented:
Hey this is a good start. Can you provide screenshots and examples of some of the features you mentioned? Otherwise this documentation is a bit sparse. Also is this a text editor I can invoke from command line? If so how?

adamspihlman commented:
@samiurkh1n I have added some more detail and examples and provided some links that help showcase Notepad++'s features. I also added instructions on how to invoke it from the command line.
